tests/sentry/models/test_groupsubscription.py

```diff
@@ -457,7 +457,10 @@ def test_simple_teams_v2(self):
 
         self._assert_subscribers_are(
             group,
-            email={self.user: GroupSubscriptionReason.implicit},
+            email={
+                self.user: GroupSubscriptionReason.implicit,
+                team: GroupSubscriptionReason.comment,
```

Comment:
We now expect the team to show up as a subscriber here (and below) but it will be filtered out [here](https://github.com/getsentry/sentry/blob/164d4111c19280ddb6d95718e7a6b446bf5746fd/src/sentry/mail/notifications.py#L116) since teams cannot receive emails